src/main/scala/com/lightbend/paradox/projectinfo/ParadoxProjectInfoPlugin.scala

```diff
@@ -44,9 +45,10 @@ object ParadoxProjectInfoPlugin extends AutoPlugin {
         Def.task {
           val s = state.value
           Seq { _: Writer.Context ⇒
-            val f           = new File((LocalRootProject / baseDirectory).value, "project/project-info.conf")
-            val extracted   = Project.extract(s)
-            val rootVersion = extracted.get(projectInfoVersion)
+            val f                  = new File((LocalRootProject / baseDirectory).value, "project/project-info.conf")
+            val extracted          = Project.extract(s)
+            val rootVersion        = projectInfoVersion.value
```

Comment:
Note that using `extracted.get` to retrieve values from sbt settings/keys is not necessary here, see https://www.scala-sbt.org/1.x/docs/Howto-Dynamic-Task.html. In fact with `readinessLevels` specifically it doesn't work because with the manual usage of `state.value` + `extracted.get` you are retrieving the `readinessLevels` before they are set in your project/sbt-plugin